docs/setup/webhooks.md

```diff
@@ -113,6 +114,12 @@ The `v2` api expects an object to be returned from the `result` variable.
 }
 ```
 
+If no HTML value is provided, `plain` will be interpreted as markdown by
+[MarkdownIt.renderInline](https://markdown-it.github.io/markdown-it/#MarkdownIt.renderInline)
+which does not support paragraph tags (`<p>`).
```

Comment:
Doesn't this just mean it doesn't wrap it in p tags. I think if you had newlines in your string it would still paragraph it?

Reply:
`.renderInline` does not convert two new lines `Foo\n\nBar` in `<p>Foo</p><p>Bar</b>` while `.render` does. One `\n` gets converted to `<br>`, `\n\n` doesn't, if I remember it correctly. This leads to `Foo\nBar\n\nFoobar` to be converted to `Foo<br>Bar Foobar` which gets rendered as

> Foo  
> Bar Foobar

which is the opposite of what you'd at least expect.

Reply:
Ah okay. I always assumed `renderInline` was `render` without the `<p>` wrapping. We could just make hookshot use `render` if you'd prefer?

Reply:
Which is what you said above in https://github.com/matrix-org/matrix-hookshot/pull/385#issuecomment-1167846005. Do you want to make the change yourself, should just be a case of fiddling with https://github.com/matrix-org/matrix-hookshot/blob/main/src/Connections/GenericHook.ts#L380 and probably updating one or two tests?